androidApp/src/main/java/dev/johnoreilly/confetti/sessiondetails/SessionDetailsView.kt

```diff
@@ -146,7 +146,7 @@ fun Chip(name: String) {
 private fun rememberShareDetails(details: SessionDetails?): () -> Unit {
     val context = LocalContext.current
 
-    return remember(details) {
+    return remember(context, details) {
```

Comment:
Out of curiosity, why is `context` needed?

Reply:
I added it because I'm capturing the context within the block - but to be fair, I don't think it is required.